moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -39,124 +39,95 @@
 #include <moveit_msgs/srv/load_map.hpp>
 #include <moveit/occupancy_map_monitor/occupancy_map.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_monitor.h>
+#include <moveit/occupancy_map_monitor/occupancy_map_monitor_middleware_handle.hpp>
 #include <boost/bind.hpp>
 
 namespace occupancy_map_monitor
 {
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.ros.occupancy_map_monitor");
-
-OccupancyMapMonitor::OccupancyMapMonitor(const rclcpp::Node::SharedPtr& node, double map_resolution)
-  : map_resolution_(map_resolution), debug_info_(false), mesh_handle_count_(0), node_(node), active_(false)
+namespace
```

Comment:
Nit: Should this namespace be outside the occupancy_map_monitor namespace for better readability? My personal preference is to have them be outside any namespace to underscore they "anonymousness".

Reply:
I'm just copying a pattern I've seen in other parts of the MoveIt code base.  I would prefer consistency, although I agree that having a separate anonymous namespace seems to be clearer.  For consistency sake I want to leave this as is.

Reply:
This is in the cpp so I would assume they wouldn't export

Reply:
But do they?

Reply:
How would I know for sure? I have just assumed that anything declared as a file global without an export in some header had only file scope.

Reply:
Well, as cppreference points out, things in anonymous namespaces can be imported.
In your gtest, I would try to access something in your anon namespace defined in your cpp

Reply:
InverseKinematics234!

Reply:
Well, as cppreference points out, things in anonymous namespaces can be imported.
In your gtest, I would try to access something in your anon namespace defined in your cpp

Reply:
cppreference:
> Even though names in an unnamed namespace may be declared with external linkage, they are never accessible from other translation units because their namespace name is unique. 

If that's what you mean by export here. But I guess you knew that before.
This only affects the individual cpp file and if anyone defines a second LOGGER there, you should change the names anyway to reflect the different meanings.

Usually I prefer anonymous namespaces **inside** file-wide namespaces to avoid having to spell out the file-wide namespace each time I need to access other symbols from there. While this is not used here, it makes some sense to keep the style consistent.

Reply:
Yeah, it didn't seem likely that anything would be exported if declared in the cpp because of the external linkage. I guess it was a bit of sticker shock since I didn't know that anon-namespace anything could be exported.

file-wide namespace?

Reply:
> file-wide namespace?

It's not a technical term. Some maintainers introduced namespace declarations that span virtually the whole file for many packages in MoveIt (e.g., [this one](https://github.com/ros-planning/moveit/commit/e01314474b2b06065daaea2a54b9d592f0f7e3c8)) to avoid the full namespace declaration for each symbol.